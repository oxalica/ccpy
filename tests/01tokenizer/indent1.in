a
  b
    c # comment before EOF