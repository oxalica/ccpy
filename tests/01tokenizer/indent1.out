Name("a")
Newline
Indent
Name("b")
Newline
Indent
Name("c")
Newline
Dedent
Dedent
End
